extensions/gemfire-modules-hibernate/src/test/java/com/gemstone/gemfire/modules/Owner.java

```diff
@@ -34,9 +34,6 @@
 /**
  * Simple JavaBean domain object representing an owner.
  * 
- * @author Ken Krebs
```

Comment:
This files looks to be copied from spring-petclinic.  We need to preserve the author tags and reapply the original copyright header.